csrankings-h.csv

```diff
@@ -288,7 +288,7 @@ Haonan Lu,University at Buffalo,https://sites.google.com/view/haonanlu/home,HeGT
 Haopeng Chen,Shanghai Jiao Tong University,http://reins.se.sjtu.edu.cn/~chenhp,DTmwMSUAAAAJ
 Haoqi Zhang,Northwestern University,http://eecs.northwestern.edu/~hq,FaGbxW0AAAAJ
 Haoyao Chen,Harbin Institute of Technology,http://faculty.hitsz.edu.cn/chenhaoyao,NOSCHOLARPAGE
-Haoyu Wang 0001,BUPT,https://howiepku.github.io,b1Oh6BQAAAAJ
+Haoyu Wang 0001,HUST,https://howiepku.github.io,b1Oh6BQAAAAJ

```

Comment:
Not clear how this school relates to CS, or if/how the faculty meet the inclusion criteria. Please re-open with an explanation.